pkg/ssh/key.go

```diff
@@ -29,9 +29,12 @@ import (
 )
 
 const (
-	privateKeyFile = "id_rsa_okteto"
-	publicKeyFile  = "id_rsa_okteto.pub"
-	bitSize        = 4096
+	privateKeyFile = "id_ecdsa_okteto"
+	publicKeyFile  = "id_ecdsa_okteto.pub"
+)
+
+var (
+	curve = elliptic.P256() // You can use P256(), P384(), or P521()
```

Comment:
Have we checked this is a valid one for the main scenario we want to fix?

Reply:
Yup, This is the list of available algos:
<img width="674" alt="Screenshot 2024-08-28 at 13 19 30" src="https://github.com/user-attachments/assets/b4062766-f1ac-4cfa-b0b1-b94972b001d7">

Reply:
would it be interesting to allow the user to choose this? just a thought, in case some algorithm is not supported we could have different algos for the user to choose ?

Reply:
I think it's not worth adding that because the maintainability vs the use is not worth. 
It's something doable but would require a new field in the manifest which is something we don't want right now. Then we should match with the algo in the remote container and on the start select the algo and create the key programatically adding more time to the start, but it's something we could reevaluate in the future.

Reply:
yup, not for this PR, just mentioning as a "future" thingy! thanks